LIPS/lip-16.md

```diff
@@ -7,9 +7,12 @@ status: Implemented
 author: Alexey Potapkin, Sam Kozin
 discussions-to: https://research.lido.fi/t/proposal-last-minute-vote-mitigation/2162/14
 created: 2022-06-01
-updated: 2022-08-26
+updated: 2025-03-21
 ---
 
+> :information_source: **LIP-16 was amended due to a [change in the voting duration](https://research.lido.fi/t/optimizing-lido-on-chain-voting-timelines-for-inclusive-governance/8265) through the [Aragon Vote #184](https://vote.lido.fi/vote/184) on March 21, 2025.** :information_source:  
+> As the number of delegates increased and different situations arose among them, the Lido DAO took steps to create a more accommodating environment to improve overall governance engagement: **the existing three-day on-chain voting duration was extended to five days**. The two-phase voting approach remains unchanged: **the Main Phase (72 hours) followed by the Objection Phase (48 hours)**.
```

Comment:
"As the number of delegates increased and different situations arose among them" sounds a bit confusing and doesn't add any new sense, lets remove

Reply:
five days -> 120 hours